go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/gogo/protobuf v1.2.0
 	github.com/golang/groupcache v0.0.0-20171101203131-84a468cf14b4
 	github.com/golang/snappy v0.0.0-20170215233205-553a64147049 // indirect
+	github.com/golang/sys v0.0.0-20190215142949-d0b11bdaac8a
```

Comment:
Where is this from anyway? I guess this might be the physical location of `golang.org/x/sys` but it should not show up here as a dependency I think?

Reply:
Idk how that happened, re-gogetting fixed it.